src/sync/incomingSyncUtils.js

```diff
@@ -931,6 +1009,94 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
       });
       break;
     }
+    case 'TemperatureLog': {
+      database.update(recordType, {
+        id: record.ID,
+        temperature: parseNumber(record.temperature),
+        timestamp: parseDate(record.date, record.time),
+        location: database.getOrCreate('Location', record.location_ID),
+        breach: database.getOrCreate('TemperatureBreach', record.breach_ID),
+      });
+      break;
+    }
+    case 'TemperatureBreach': {
+      database.update(recordType, {
+        id: record.ID,
+        startTimestamp: parseDate(record.start_date, record.start_time),
+        endTimestamp: parseDate(record.end_date, record.end_time),
+        location: database.getOrCreate('Location', record.location_ID),
+      });
+      break;
+    }
+    case 'TemperatureBreachConfiguration': {
+      database.update(recordType, {
+        id: record.ID,
+        minimumTemperature: parseNumber(record.minimum_temperature),
+        maximumTemperature: parseNumber(record.maximum_temperature),
+        duration: parseNumber(record.duration),
+        description: record.description,
+        colour: record.colour,
+      });
+      break;
+    }
+    case 'LocationMovement': {
+      database.update(recordType, {
+        id: record.ID,
+        location: database.getOrCreate('Location', record.location_ID),
+        itemBatch: database.getOrCreate('Location', record.item_line_ID),
+        enterTimestamp: parseDate(record.enter_date, record.enter_time),
+        exitTimestamp: parseDate(record.exit_date, record.exit_time),
+      });
+      break;
+    }
+    case 'VaccineVialMonitorStatus': {
+      database.update(recordType, {
+        id: record.ID,
+        description: record.description,
+        code: record.code,
+        level: parseNumber(record.level),
+        isActive: parseBoolean(record.boolean),
```

Comment:
there's `record.boolean` again. Is it ok?

Reply:
spooky mistake! 285b03f58cda4382025894656533f442ab413c02